Dockerfile

```diff
@@ -8,7 +8,8 @@ MAINTAINER opentrials
 
 USER root
 RUN apt-get update -yqq && \
-    apt-get install -yqq python-pip
+    apt-get install -yqq python-pip && \
+    apt-get install -yqq postgresql-client
```

Comment:
Be aware that in Debian Jessie (our upstream's base image) the client version in 9.4. AFAIK there's 9.5 backported via `jessie-backports` repository, but I might be wrong.
Not sure if client version mismatch would affect our ability to dump, in this case at least. Might be just fine.